wger/nutrition/views/meal.py

```diff
@@ -23,8 +26,11 @@
 
 from django.views.generic import CreateView, UpdateView
 
-from wger.nutrition.models import NutritionPlan, Meal
+from wger.nutrition.models import NutritionPlan, Meal, MealItem
 from wger.utils.generic_views import WgerFormMixin
+# from wger.nutrition.views.meal_item import MealItemCreateView
```

Comment:
Kindly remove this commented out line

Reply:
Thank you for the feedback. This has been rectified